cucumber/authenticators_jwt/features/support/jwt_jwks_helper.rb

```diff
@@ -78,6 +79,40 @@ def issue_jwt_token_with_jwk(token_body, file_name, algorithm = Algorithms::RS25
     )
   end
 
+  def issue_jwt_token_with_x5c(token_body, algorithm = Algorithms::RS256)
+    cert = self_signed_certificate(rsa_keys[@default_file_name])
```

Comment:
I guess that we should use two different RSA keys like in jku/jkw tests. One RSA is behind JWKS endpoint and one another signs the token and certificate.

Reply:
why shouldnt we use the same key? it doesnt affect anything as far as i can tell

Reply:
The aim of the test is to validate that the token is not self-contained. Using the same keys logically negates the aim, from my point of view.